README.md

```diff
@@ -4,12 +4,16 @@
 [![License](https://img.shields.io/badge/license-LGPLv2.0%2B-purple.svg)](https://www.gnu.org/licenses/old-licenses/lgpl-2.0.en.html)
 [![ContentDB](https://content.minetest.net/packages/mt-mods/wrench/shields/downloads/)](https://content.minetest.net/packages/mt-mods/wrench/)
 
-The wrench tool allows you pickup nodes with inventories or other metadata, such as chests, signs, drawers, machines, etc, and transport them intact to another location.
+The wrench tool allows you pick up nodes with inventories or other metadata, such as chests, signs, drawers, machines, etc, and transport them intact to another location.
```

Comment:
... allows you _**to**_ pick up nodes ...

It's correct in the mod.conf.
Personally I'd also drop the "you" or replace it with "player" but that isn't important, the missing "to" is.

Reply:
Nice catch :)